docs/troubleshoot/app-framework/appserver-error-codes.md

```diff
@@ -2053,303 +2053,271 @@ The following error message codes may appear on the app-server log. Use the foll
 
   **Reason:**
 
-  TODO
+  The server throws a warning when the CLI argument is missing a name, or has an unsupported type (supported types: 1 - _flag_, 2 - _value_, 3 - _json_).
 
   **Action:**
 
-  TODO
+  Check any missing argument or unsupported argument.
+
 
 
+### ZWED0154W
 
-### ZWED0158W":"%s
+  WARNING: Unrecognized command: _args_
 
   **Reason:**
 
-  TODO
+  Throws warning when _args_ is unrecognized.
 
   **Action:**
 
-  TODO
+  Check the command once again or check if the specified command is interpreted as intended.
+
 
 
+### ZWED0155W
 
-### ZWED0159W":"Plugin (%s) loading failed. Message: "%s" Successful: %s% (%s/%s) Attempted: %s% (%s/%s)
+   _ErrorMsg_
 
   **Reason:**
 
-  TODO
+  Server throws 500 code with _ErrorMsg_.
 
   **Action:**
 
-  TODO
+  Go through the _ErrorMsg_ for context on what to debug.
 
 
 
-### ZWED0160W":"Master cluster not found to save storage for id: %s, value: %s
+### ZWED0156W
+
+  1 function initLoggerMessages - ERROR - _Error_
 
   **Reason:**
 
-  TODO
+  Attempt to get log message for a language a user may have specified, has failed with _Error_.
 
   **Action:**
 
-  TODO
+  Go through the Error for details on what to debug.
+
 
 
+### ZWED0157W
 
-### ZWED0161W":"Master cluster not found to save storage
+  2 function initLoggerMessages - ERROR - _Error_.
 
   **Reason:**
 
-  TODO
+  Attempt to get log message for English has failed with _Error_.
 
   **Action:**
 
-  TODO
+  Go through the Error for details on what to debug.
+
 
 
+### ZWED0158W
 
-### ZWED0162W":"Master cluster not found to add storage
+   _ErrorMsg_
 
   **Reason:**
 
-  TODO
+  Server throws 500 code with _ErrorMsg_.
 
   **Action:**
 
-  TODO
+  Go through the _ErrorMsg_ for details on what to debug.
 
 
 
-### ZWED0163W":"Master cluster was not found to delete storage for id: %s
+### ZWED0159W
+
+  Plugin (_PluginIdentifier_) loading failed. Message: "_errorMessage_" Successful: _pluginsLoaded_% (_pluginsLoaded_/_eventCount_) Attempted: _pluginCount_% (_pluginCount_/_eventCount_)
 
   **Reason:**
 
-  TODO
+  Plugin with _pluginId_ loaded failed with _errorMessage_. The server attempted to load a total of _pluginCount_ with _pluginsLoaded_ plugins already successfully loaded.
 
   **Action:**
 
-  TODO
+  Review _errorMessage_ to see if there is something you can do to fix the error. You may need to contact the plugin developer to find a solution. If you do not need this plugin, it is OK to continue.
+
 
 
+### ZWED0166W
 
-### ZWED0165W":"Unable to retrieve storage data from cluster promise %s
+  Error updating the storage: _Error_
 
   **Reason:**
 
-  TODO
+  Throws warning _Error_ when it faced error while updating the storage.
 
   **Action:**
 
-  TODO
+  Contact support if _Error_ is not clear.
 
 
 
-### ZWED0166W":"Error updating the storage: %s
+### ZWED0167W
+
+  Error adding to the storage: _storageTimeout_
```

Comment:
Not true

`if (e.startsWith(ERROR_TIMEOUT_CALL)) {
          clusterLogger.severe("ZWED0144E - Cluster storage object timeout." +
                               "\nYou may change the value of '" + storageTimeout + "' ms by setting 'node.cluster.storageTimeout' within the config.");
        } else {
          clusterLogger.warn("ZWED0167W - Error adding to the storage: " + e);
        }`
        
        in the if case yes, but yours isnt, it's the else case. Please revise this message, it's an error for all non-timeout issues